frontend/bank-transfer-ui/src/app/main/transfer-list/transfer-list.component.html

```diff
@@ -1,10 +1,96 @@
-
-<div class="content">
-
+<div class="wrapper">
   <div class="container">
+    <div class="content">
 
-    <h1>Sample text</h1>
+      <h3 class="display-4 mb-3">Transfer history</h3>
+      <hr class="mb-3">
+      <div class="my-3">
+        <button type="button" class="btn btn-secondary btn-custom btn-lg" (click)="open(content)">Filter options
+        </button>
+      </div>
+      <div *ngFor="let tran of transfer | groupBy:'createTime'">
+        <p class="mb-1 mt-4">{{tran?.key | date:'fullDate'}}</p>
+        <ul class="list-group">
+          <li class="list-group-item list-group-item-dark list-group-item-custom" *ngFor="let list of tran.value">
+            <div class="row">
+              <div class="col-md-2 col-sm-3">{{list?.status}}</div>
+              <div class="col-md-3 col-sm-3">{{list?.title}}</div>
+              <div class="col-md-4 col-sm-auto">{{list?.accountNo}}</div>
+              <div class="col-md-2 col-sm-auto">{{list?.value | currency:'PLN':'code':'1.2-2':'pl'}}</div>
+            </div>
+          </li>
+        </ul>
+      </div>
 
+      <div class="search-results"
+           infiniteScroll
+           [infiniteScrollDistance]="85"
+           [infiniteScrollThrottle]="1300"
+           (scrolled)="onScroll()">
+      </div>
+      <ngx-spinner
```

Comment:
Masz różne konwencje układania HTML, raz atrybuty są w jednej linii, raz pierwszy atrybut jest obok tagu, a reszta pod nim, a tutaj wszystkie atrybuty zaczynają się pod tagiem. Warto uspójnić to - zwłaszcza jak pracujesz z kimś w projekcie